static/js/unread_ui.js

```diff
@@ -80,32 +78,12 @@ function consider_bankruptcy() {
 
     const now = new XDate(true).getTime() / 1000;
     if (page_params.unread_msgs.count > 500 &&
-            now - page_params.furthest_read_time > 60 * 60 * 24 * 2) { // 2 days.
-        const rendered_modal = render_bankruptcy_modal({
-            unread_count: page_params.unread_msgs.count});
-        $('#bankruptcy-unread-count').html(rendered_modal);
-        $('#bankruptcy').modal('show');
-    } else {
-        exports.enable();
+        now - page_params.furthest_read_time > 60 * 60 * 24 * 2) { // 2 days.
+        return true;
```

Comment:
I think the bankruptcy whole block should be moved to its own function outside `initialize`; generally it's bad form for readability to call an `initialize` function more than once.

We can call it `unread_ui.should_display_bankruptcy_banner()`.

Reply:
And I think we probably can just make initialize just call `exports.enable()` unconditionally as its full implementation, since without the modal structure, there's no reason.  

And then we should add follow-up commits removing some of the unreads-not-enabled infrastructure that this had had.